### PR TITLE
feat: lpi2

### PR DESCRIPTION
**Pull request by sharafian**

### CODE REVIEW

**Comment by michielbdejong** [comment]
I think I'll skip lpi2 and wait until lpi3 comes out ;)

File: test/lpi2Spec.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    const expiry = new Date(Date.now() + 10000)
+    this.fulfillment = 'gHJ2QeIZpstXaGZVCSq4d3vkrMSChNYKriefys3KMtI'
+    this.condition = base64url(crypto
+      .createHash('sha256')
+      .update(this.fulfillment, 'base64')
+      .digest())
+
+    this.transfer = {
+      id: uuid(),
```
Comment (michielbdejong) [inline test/lpi2Spec.js:59]: Isn't the role of transaction id taken over by the promise that sendTransfer returns?

Comment (sharafian) [reply test/lpi2Spec.js:59]: Good point. We can still keep it in BTP (to keep this compatible with current BTP) but the LPI has no reason to expose it.

**Comment by michielbdejong** [comment]
We could make this a major version, and then everybody who still wants to use this plugin the normal way needs to require a version of it that no longer receives bugfixes.

Would it also be possible to include a translation util (or provide that as a separate wrapper module), so that the latest updated plugin code can still be used by its current users, behind a normal lpi version 1 interface?

**Pull request closed without merging**